_i18n/en.yml

```
@@ -59,7 +59,14 @@ contribute:
     text: "You are good at breaking stuff? Become a beta tester."
   develop:
     title: "Develop"
-    text: "You can program in Java? Help test pull requests or submit new features or bug fixes on GitHub."
+    text: "You write code? Help test pull requests or submit new features or bug fixes for the app or website."
+  promote:
+    title: "Promote"
+    text: "You draft catchy tweets, interesting blogs, or wonderful graphics? Help spread the word."
+  design:
+    title: "Design"
+    text: "You turn complex requests in understandable user stories or wireframes? Help tackle these issues."
+  donate: "Or, if you can't contribute in any of the ways listed above, you can always make a <a href="https://opencollective.com/antennapod" target="_blank">donation</a>."
```

I would maybe also explicitly note that help is more appreciated than money.

---

I would say that 'if you can't contribute in any other way' (linguistically) already indicates that money is of secondary importance. I wouldn't _add_ it, but we can replace it with the following (though it kinda ends on a negative note):
```suggestion
  donate: "Contributions in any of the ways listed above are appreciated more, but you can also make a <a href="https://opencollective.com/antennapod" target="_blank">donation</a>."
```

---

You are right, that sounds pretty negative. What do you think about replacing "always" with "also"? Maybe the text can also be changed to gray color to indicate that it really isn't the main thing we need.